generator/sbpg/targets/resources/sbp_base.tex

```diff
@@ -32,6 +32,7 @@
 \numberwithin{table}{subsection}
 \numberwithin{field}{subsection}
 
+\renewcommand{\version}{(((version)))}
```

Comment:
I'd just remove this and use the macro input directly everywhere - It's confusing now because you're using the macro in some spots and the function in others.